hazenlib/__init__.py

```diff
@@ -196,7 +196,8 @@ def main():
     if arguments['snr'] or arguments['<task>'] == 'snr':
         selected_task = 'snr'
         task = init_task(selected_task, files, report, report_dir,
-                         measured_slice_width=arguments['--measured_slice_width'])
+                         measured_slice_width=arguments['--measured_slice_width'],
+                         coil = arguments['--coil'])
```

Comment:
when you specify a value for an input to a function, usually there are no spaces around the '=' sign